Purchases/SwiftSources/IntroEligibilityCalculator.swift

```diff
@@ -11,17 +11,17 @@ import StoreKit
 
 public class IntroEligibilityCalculator: NSObject {
     private let productsManager: ProductsManager
-    private let localReceiptParser: LocalReceiptParser
```

Comment:
It's not technically `local` if it receives `NSData`, the data could come from anywhere.